### ISSUE TITLE
Better interface for move/rotate of a group/component

### ISSUE THREAD

**Issue by fijal** [lightscapevr/vr-sketch-4#16]
It would be really useful if we can have a ghost 3D model of group/component when moving it or rotating (or scaling). The boundaries are often arbitrary and just because you can move a boundary and snatch it, does not mean the right thing happens for the whole component

**Comment by arigo** [comment]
I think the boundaries are not used for snatching: only the point where you click-started is, and it can be anywhere inside the group (or even outside if you first selected the group).  So it should be only a displaying issue: instead of seeing the whole group moving, we see only its bounding box.

**Comment by fijal** [comment]
Right, let's start with the display issue and then let's see if we're happy with the result
